x-pack/plugins/ingest_manager/public/applications/ingest_manager/sections/fleet/setup_page/index.tsx

```diff
@@ -87,23 +87,23 @@ export const SetupPage: React.FunctionComponent<{
               <h2>
                 <FormattedMessage
                   id="xpack.ingestManager.setupPage.enableTitle"
-                  defaultMessage="Enable Fleet"
+                  defaultMessage="Enable central management for Elastic Agents"
                 />
               </h2>
             </EuiTitle>
             <EuiSpacer size="xl" />
             <EuiText color="subdued">
               <FormattedMessage
                 id="xpack.ingestManager.setupPage.enableText"
-                defaultMessage="Fleet requires an Elastic user who can create API keys and write to logs-* and metrics-*."
+                defaultMessage="Central management for Elastic Agents requires an Elastic user who can create API keys and write to logs-* and metrics-*."
               />
             </EuiText>
             <EuiSpacer size="l" />
             <EuiForm>
               <EuiButton onClick={onSubmit} fill isLoading={isFormLoading} type="submit">
                 <FormattedMessage
-                  id="xpack.ingestManager.setupPage.enableFleet"
-                  defaultMessage="Create user and enable Fleet"
+                  id="xpack.ingestManager.setupPage.enableCentralManagement"
+                  defaultMessage="Create user and enable Agents"
```

Comment:
```suggestion
                  defaultMessage="Create user and enable central management"
```

enable Agents sounds weird to me but I'll defer to @dedemorton and @gchaps

Reply:
Brainstormed with @dedemorton.  Instead of "enable", you could use "turn on".  Also how about considering a different break in the title.

Turn on central management
for Elastic Agents